app.json

```diff
@@ -56,6 +56,9 @@
     "CYBERSOURCE_SECURITY_KEY": {
       "description": "CyberSource API key"
     },
+    "CYBERSOURCE_TRANSACTION_KEY": {
```

Comment:
This is adding this key again, which was recently removed in https://github.com/mitodl/micromasters/pull/4054/files, is this intentional?